lib/lib-storage/src/Upload.ts

```diff
@@ -364,6 +369,12 @@ export class Upload extends EventEmitter {
 
     let result;
     if (this.isMultiPart) {
+      if (this.totalBytes && this.uploadedParts.length !== this.expectedPartsCount) {
+        throw new Error(
+          `Expected ${this.expectedPartsCount} number of parts but uploaded only ${this.uploadedParts.length} parts`
```

Comment:
```suggestion
          `Expected ${this.expectedPartsCount} part(s) but uploaded ${this.uploadedParts.length} part(s).`
```

Can't say "only" because the check is for inequality. The uploaded parts count could be higher.